contracts/BatchExchangeViewer.sol

```diff
@@ -8,6 +8,7 @@ contract BatchExchangeViewer {
     using BytesLib for bytes;
 
     uint8 public constant AUCTION_ELEMENT_WIDTH = 112;
+    uint16[] public ALL_TOKEN_FILTER;
```

Comment:
Is this being used anywhere?

Reply:
It's not. I wasn't able to create an empty memory array in solidity so I exposed it to be used as BatchExchangeViewer.ALL_TOKEN_FILTER in other contracts. But theoretically the could also declare it. Thought it could serve as documentation too, but happy to delete.

Reply:
Oh I get it. I say we can leave it in then. Maybe a doc comment to explain what it is for.